airflow/providers/amazon/aws/operators/appflow.py

```diff
@@ -0,0 +1,545 @@
+            destinationFlowConfigList=response["destinationFlowConfigList"],
+            sourceFlowConfig=response["sourceFlowConfig"],
+            triggerConfig=trigger_config,
+            description=response.get("description", "Flow description."),
+            tasks=tasks,
+        )
+
+    def _add_filter(self, connector_type: str, tasks: List["TaskTypeDef"]) -> None:  # Interface
+        pass
+
+    def _run_flow(self, context) -> str:
+        ts_before: datetime = datetime.now(timezone.utc)
+        sleep(EVENTUAL_CONSISTENCY_OFFSET)
+        response = self.hook.conn.start_flow(flowName=self.name)
+        task_instance = context["task_instance"]
+        task_instance.xcom_push("execution_id", response["executionId"])
+        self.log.info("executionId: %s", response["executionId"])
+
+        response = self.hook.conn.describe_flow(flowName=self.name)
+
+        # Wait Appflow eventual consistence
+        self.log.info("Waiting Appflow eventual consistence...")
+        while (
+            response.get("lastRunExecutionDetails", {}).get(
+                "mostRecentExecutionTime", datetime(1970, 1, 1, tzinfo=timezone.utc)
+            )
+            < ts_before
+        ):
+            sleep(EVENTUAL_CONSISTENCY_POLLING)
+            response = self.hook.conn.describe_flow(flowName=self.name)
+
+        # Wait flow stops
+        self.log.info("Waiting flow run...")
+        while (
+            "mostRecentExecutionStatus" not in response["lastRunExecutionDetails"]
+            or response["lastRunExecutionDetails"]["mostRecentExecutionStatus"] == "InProgress"
+        ):
+            sleep(self.poll_interval)
+            response = self.hook.conn.describe_flow(flowName=self.name)
+
+        self.log.info("lastRunExecutionDetails: %s", response["lastRunExecutionDetails"])
+
+        if response["lastRunExecutionDetails"]["mostRecentExecutionStatus"] == "Error":
+            raise Exception(f"Flow error:\n{json.dumps(response, default=str)}")
+
+        return response["lastRunExecutionDetails"]["mostRecentExecutionStatus"]
+
+
+class AppflowRunOperator(AppflowBaseOperator):
+    """
+    Execute a Appflow run with filters as is.
+
+    .. seealso::
+        For more information on how to use this operator, take a look at the guide:
+        :ref:`howto/operator:AppflowRunOperator`
+
+    :param source: The source name (e.g. salesforce, zendesk)
+    :param name: The flow name
+    :param poll_interval: how often in seconds to check the query status
+    :param aws_conn_id: aws connection to use
+    :param region: aws region to use
+    """
+
+    def __init__(
+        self,
+        source: str,
+        name: str,
+        poll_interval: int = 20,
+        aws_conn_id: Optional[str] = "aws_default",
+        region: Optional[str] = None,
+        **kwargs,
+    ) -> None:
+        if source not in {"salesforce", "zendesk"}:
+            raise AirflowException(f"Source {source} is not supported for AppflowRunOperator!")
```

Comment:
```suggestion
        if source not in SUPPORTED_SOURCES:
            raise ValueError(f"Source {source!r} is not supported for AppflowRunOperator!")
```
Same comment as above re: the exception type.